packages/ui/src/Layout/Layout.stories.tsx

```diff
@@ -53,11 +54,17 @@ const user = {
     "https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80",
 };
 export const Default = (): JSX.Element => (
-  <Layout {...{ user }}>{dummyContent}</Layout>
+  <BrowserRouter>
```

Comment:
It would be cleaner to add react router add on to storybook (I’ll take care of that)